scripts/meta/west/log.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Open Source Foundries Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''Logging module for west
```

Comment:
Doesn't Python include the concept of "loggers"? do we need our own module for this?

Reply:
I think I may need to rename this module `msg` (for "message") if this is too confusing, but the idea is precisely to abstract out whether the logging module is used, or stdout / stderr are being used directly, etc.

Reply:
OK, that makes sense.